axis/door_control.py

```diff
@@ -0,0 +1,188 @@
+"""Door Control API.
+
+The Axis Door control API makes it possible to control the behavior and functionality
+of physical access controls in the Axis devices (e.g. A1001, A1601)
+"""
+
+from .api import APIItem, APIItems
+
+URL = "/vapix/doorcontrol"
+
+API_DISCOVERY_ID = "door-control"
+API_VERSION = "1.0"
+
+CAPABILITY_ACCESS = "access"
+CAPABILITY_LOCK = "lock"
+CAPABILITY_UNLOCK = "unlock"
+CAPABILITY_BLOCK = "block"
+CAPABILITY_DOUBLE_LOCK = "doublelock"
+CAPABILITY_LOCK_DOWN = "lockdown"
+CAPABILITY_LOCK_OPEN = "lockopen"
+CAPABILITY_DOOR_MONITOR = "doormonitor"
+CAPABILITY_LOCK_MONITOR = "lockmonitor"
+CAPABILITY_DOUBLE_LOCK_MONITOR = "doublelockmonitor"
+CAPABILITY_ALARM = "alarm"
+CAPABILITY_TAMPER = "tamper"
+CAPABILITY_WARNING = "warning"
+CAPABILITY_CONFIGURABLE = "configurable"
+
+SUPPORTED_CAPABILITIES = (
+    CAPABILITY_ACCESS,
+    CAPABILITY_LOCK,
+    CAPABILITY_UNLOCK,
+    CAPABILITY_BLOCK,
+    CAPABILITY_DOUBLE_LOCK,
+    CAPABILITY_LOCK_DOWN,
+    CAPABILITY_LOCK_OPEN,
+    CAPABILITY_DOOR_MONITOR,
+    CAPABILITY_LOCK_MONITOR,
+    CAPABILITY_DOUBLE_LOCK_MONITOR,
+    CAPABILITY_ALARM,
+    CAPABILITY_TAMPER,
+    CAPABILITY_WARNING,
+    CAPABILITY_CONFIGURABLE
+)
+
+
+class DoorControl(APIItems):
+    """Door control for Axis devices."""
+
+    def __init__(self, request: object) -> None:
+        super().__init__({}, request, URL, Door)
+
+    # TODO: Question: Is this used to get status information for the door?  Or to update the object properties?
```

Comment:
Whats the difference?

Reply:
That was just an early musing before I got into a better understanding of what was going on.  Will remove.